projectforum/settings.py

```diff
@@ -114,11 +114,11 @@
 DATABASES = {}
 DATABASES['default'] =  dj_database_url.config(default='postgres://:@localhost/projectforum_db')
 
-TEST_DATABASES = {
-    'default': dj_database_url.config(env='TEST_DATABASE_URL')
-}
+# TEST_DATABASES = {
```

Comment:
We need to keep the commited version of settings.py uncommented out so we can run remotely.